### PR TITLE
Added Tables to Iago

### PR DESCRIPTION
**Pull request by YogatBear**
Added "Table" section to settings for adjusting rows and columns. Enabling headers adds another set of shorter cells to the left or above the cells for rows or columns respectively.

Added table to the "add" dropdown menu.

### CODE REVIEW

**Comment by raisjn** [comment]
awesome, thank you! can you take a screenshot to show the settings dialog and the output? 

code wise, the .h files in shared/ are not needed

File: Agent
Comment (Eeems) [inline Agent:1]: What's this file for?

Comment (raisjn) [reply Agent:1]: vscode settings, i imagine. i'll remove it

Comment (Eeems) [reply Agent:1]: Oh, I was asking about `Agent` not `.vscode`

Comment (raisjn) [reply Agent:1]: what the. I didn't see this file before

**Comment by YogatBear** [comment]
Some screenshots in case you still need them:
fb
fb1

**Pull request merged**